CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+- Move `internal/cmd` to `cmd/` so that it can be depended on.
```

Comment:
`### Changed`

Reply:
Good catch - using `Added` since this is more additive from a user perspective. `Changed` is a change in behavior and sometimes implies a breaking change.